index.test.js

```diff
@@ -11,18 +11,21 @@ test('test runs', () => {
     process.env['GITHUB_REPOSITORY'] = 'ru2-apic-discovery-action';
     process.env['GITHUB_WORKSPACE'] = '/home/ruairi/git/misc/apic-discovery-action';
 
-    process.env['INPUT_API_HOST'] = 'd-h01.apiconnect.dev.automation.ibm.com';
-    process.env['INPUT_API_KEY'] = '';
-    process.env['INPUT_PROVIDER_ORG'] = 'ruairi_h01_b';
+    process.env['INPUT_API-HOST'] = 'd-h01.apiconnect.dev.automation.ibm.com';
+    process.env['INPUT_API-KEY'] = '';
+    process.env['INPUT_PROVIDER-ORG'] = 'ruairi_h01_b';
     // process.env['INPUT_GIT_DIFF'] = 'APIfolder/gmail-api.json mit-api.json new-api.yaml';
 
-    process.env['INPUT_API_FILES'] = [ 'APIfolder/gmail-api.json', 'APIfiles/mit-api.json' ];
-    // process.env['INPUT_API_FILES'] = [ 'gmail-api-2.json' ];
-    // process.env['INPUT_API_FOLDERS'] = [ 'APIfiles' ];
-    // process.env['INPUT_API_FOLDERS'] = [ 'APIfiles', 'APIfolder' ];
+    process.env['INPUT_API-FILES'] = [ 'APIfolder/gmail-api.json', 'APIfiles/mit-api.json' ];
+    // process.env['INPUT_API-FILES'] = [ 'gmail-api-2.json' ];
+    // process.env['INPUT_API-FOLDERS'] = [ 'APIfiles' ];
+    // process.env['INPUT_API-FOLDERS'] = [ 'APIfiles', 'APIfolder' ];
     process.env['INPUT_RESYNC_CHECK'] = true;
 
     const ip = path.join(__dirname, 'index.js');
+    console.log(process.env);
```

Comment:
@NiraimathiGunalan , I think you need to add these console debug statements into the main index.js and discovery-client code to understand what is actually throwing the exception